.php_cs.dist

```diff
@@ -16,7 +16,7 @@ $finder = PhpCsFixer\Finder::create()
     ->append([__DIR__.'/php-cs-fixer'])
 ;
 
-$config = PhpCsFixer\Config::create()
+$config = (new \PhpCsFixer\Config())
```

Comment:
The leading `\` is superfluous. Wondering if that's the reason fabbot fails.

Reply:
Oh crap, you're right. Missed copy/paste. I'll update that and see where it goes.

Reply:
Hmm. Still fails. It seems like fabbot.io has a config somewhere that uses a whiltelist system for allowed called/included files. Need to figure out where. ^^

Reply:
So far, I don't see where this config lies. Maybe it's on fabbot.io side. @fabpot, got any clue? :(

Reply:
Update: fixed the tests, seems so. There was a very well hidden call deep within tests, didn't pay attention to this one.

Still, no clue about fabbot.io config. Maybe we need to summon the Fabien / @fabpot on this one. This looks beyond my powers.

Reply:
I've just fixed the sandbox, fabbot is now green :)

Reply:
Thanks A LOT @fabpot, I was almost getting mad trying to figure out where the PHPSandbox was :)